app/routes/friends/new.js

```diff
@@ -1,15 +1,16 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  model: function() {
+  model() {
     return this.store.createRecord('friend');
   },
 
   actions: {
-    save: function() {
+    save() {
       return true;
     },
-    cancel: function() {
+
+    cancel() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).